### ISSUE TITLE
(de)Buffs not Loading

### ISSUE THREAD

**Issue by cb0wman** [tukui-org/ElvUI#1166]
### Troubleshooting Steps

![Screenshot 2024-02-06 173059](https://github.com/tukui-org/ElvUI/assets/14238546/561682ea-555b-4181-b824-573cbd6583a0)

### Describe the expected behavior and what actually happened?

Expect to see ElvUI (de-)buffs but instead see only Blizzard default (de-)buffs.

### Reproducing the issue.

open elvui options.  goto Buffs and Debuffs tab and adjust Size or another slider / configuration variable.

### Lua Errors

```shell
Date: 2024-02-06 17:21:47
ID: 1
Error occured in: Global
Count: 1
Message: Interface/AddOns/ElvUI/Core/Modules/Auras/Auras.lua line 450:
   attempt to index local 'header' (a nil value)
Debug:
   [string "@Interface/AddOns/ElvUI/Core/Modules/Auras/Auras.lua"]:450: UpdateHeader()
   [string "@Interface/AddOns/ElvUI_Options/Core/Auras.lua"]:79:
      Interface/AddOns/ElvUI_Options/Core/Auras.lua:79
   [string "=[C]"]: ?
   [string "@Interface/AddOns/ElvUI_Libraries/Core/Ace3/AceConfig-3.0/AceConfigDialog-3.0/AceConfigDialog-3.0.lua"]:45:
      ...nfig-3.0/AceConfigDialog-3.0/AceConfigDialog-3.0.lua:43
   [string "@Interface/AddOns/ElvUI_Libraries/Core/Ace3/AceConfig-3.0/AceConfigDialog-3.0/AceConfigDialog-3.0.lua"]:869:
      ...nfig-3.0/AceConfigDialog-3.0/AceConfigDialog-3.0.lua:691
   [string "@Interface/AddOns/ElvUI_Libraries/Core/Ace3/AceConfig-3.0/AceConfigDialog-3.0/AceConfigDialog-3.0.lua"]:926:
      ...nfig-3.0/AceConfigDialog-3.0/AceConfigDialog-3.0.lua:909
   [string "=[C]"]: ?
   [string "@Interface/AddOns/ElvUI_Libraries/Core/Ace3/AceGUI-3.0/AceGUI-3.0.lua"]:66:
      .../ElvUI_Libraries/Core/Ace3/AceGUI-3.0/AceGUI-3.0.lua:64
   [string "@Interface/AddOns/ElvUI_Libraries/Core/Ace3/AceGUI-3.0/AceGUI-3.0.lua"]:300: Fire()
   [string "@Interface/AddOns/ElvUI_Libraries/Core/Ace3/AceGUI-3.0/widgets/AceGUIWidget-Slider-ElvUI.lua"]:65:
      ...ce3/AceGUI-3.0/widgets/AceGUIWidget-Slider-ElvUI.lua:56
Locals:
self = <table> {
 SetDefaultModuleLibraries = <function> defined @Interface/AddOns/Details/Libs/AceAddon-3.0/AceAddon-3.0.lua:363
 Header_OnEvent = <function> defined @Interface/AddOns/ElvUI/Core/Modules/Auras/Auras.lua:401
 Enable = <function> defined @Interface/AddOns/Details/Libs/AceAddon-3.0/AceAddon-3.0.lua:290
 NewModule = <function> defined @Interface/AddOns/Details/Libs/AceAddon-3.0/AceAddon-3.0.lua:231
 baseName = "ElvUI"
 EnableModule = <function> defined @Interface/AddOns/Details/Libs/AceAddon-3.0/AceAddon-3.0.lua:328
 modules = <table> {
 }
 GetModule = <function> defined @Interface/AddOns/Details/Libs/AceAddon-3.0/AceAddon-3.0.lua:206
 IterateEmbeds = <function> defined @Interface/AddOns/Details/Libs/AceAddon-3.0/AceAddon-3.0.lua:438
 UpdateTempEnchant = <function> defined @Interface/AddOns/ElvUI/Core/Modules/Auras/Auras.lua:294
 SecureHookScript = <function> defined @Interface/AddOns/ElvUI_Libraries/Core/Ace3/AceHook-3.0/AceHook-3.0.lua:409
 defaultModuleLibraries = <table> {
 }
 Button_OnLeave = <function> defined @Interface/AddOns/ElvUI/Core/Modules/Auras/Auras.lua:331
 CreateAuraHeader = <function> defined @Interface/AddOns/ElvUI/Core/Modules/Auras/Auras.lua:505
 UpdateHeader = <function> defined @Interface/AddOns/ElvUI/Core/Modules/Auras/Auras.lua:447
 RawHookScript = <function> defined @Interface/AddOns/ElvUI_Libraries/Core/Ace3/AceHook-3.0/AceHook-3.0.lua:394
 Header_OnUpdate = <function> defined @Interface/AddOns/ElvUI/Core/Modules/Auras/Auras.lua:413
 Button_OnAttributeChanged = <function> defined @Interface/AddOns/ElvUI/Core/Modules/Auras/Auras.lua:387
 Button_OnEnter = <function> defined @Interface/AddOns/ElvUI/Core/Modules/Auras/Auras.lua:335
 SendMessage = <function> defined @Interface/AddOns/SlideBar/Libs/LibDataBroker/CallbackHandler/CallbackHandler-1.0.lua:49
 GetName = <function> defined @Interface/AddOns/Details/Libs/AceAddon-3.0/AceAddon-3.0.lua:275
 UpdateTime = <function> defined @Interface/AddOns/ElvUI/Core/Modules/Auras/Auras.lua:356
 Button_OnHide = <function> defined @Interface/AddOns/ElvUI/Core/Modules/Auras/Auras.lua:348
 Button_OnShow = <function> defined @Interface/AddOns/ElvUI/Core/Modules/Auras/Auras.lua:341
 name = "ElvUI_Auras"
 Initialize = <function> defined @Interface/AddOns/ElvUI/Core/Modules/Auras/Auras.lua:550
 IsEnabled = <function> defined @Interface/AddOns/Details/Libs/AceAddon-3.0/AceAddon-3.0.lua:447
 orderedModules = <table> {
 }
 SetTooltip = <function> defined @Interface/AddOns/ElvUI/Core/Modules/Auras/Auras.lua:323
 DisableModule = <function> defined @Interface/AddOns/Details/Libs/AceAddon-3.0/AceAddon-3.0.lua:346
 SetAuraTime = <function> defined @Interface/AddOns/ElvUI/Core/Modules/Auras/Auras.lua:228
 UpdateIcon = <function> defined @Interface/AddOns/ElvUI/Core/Modules/Auras/Auras.lua:199
 SetDefaultModulePrototype = <function> defined @Interface/AddOns/Details/Libs/AceAddon-3.0/AceAddon-3.0.lua:405
 UpdateAura = <function> defined @Interface/AddOns/ElvUI/Core/Modules/Auras/Auras.lua:269
 RegisterMessage = <function> defined @Interface/AddOns/SlideBar/Libs/LibDataBroker/CallbackHandler/CallbackHandler-1.0.lua:80
 HookScript = <function> defined @Interface/AddOns/ElvUI_Libraries/Core/Ace3/AceHook-3.0/AceHook-3.0.lua:365
 ClearAuraTime = <function> defined @Interface/AddOns/ElvUI/Core/Modules/Auras/Auras.lua:247
 UnregisterMessage = <function> defined @Interface/AddOns/SlideBar/Libs/LibDataBroker/CallbackHandler/CallbackHandler-1.0.lua:143
 IsHooked = <function> defined @Interface/AddOns/ElvUI_Libraries/Core/Ace3/AceHook-3.0/AceHook-3.0.lua:492
 CreateIcon = <function> defined @Interface/AddOns/ElvUI/Core/Modules/Auras/Auras.lua:122
 UnregisterAllEvents = <function> defined @Interface/AddOns/SlideBar/Libs/LibDataBroker/CallbackHandler/CallbackHandler-1.0.lua:164
 IsModule = <function> defined @Interface/AddOns/Details/Libs/AceAddon-3.0/AceAddon-3.0.lua:213
 defaultModuleState = true
 SetDefaultModuleState = <function> defined @Interface/AddOns/Details/Libs/AceAddon-3.0/AceAddon-3.0.lua:383
 MasqueData = <function> defined @Interface/AddOns/ElvUI/Core/Modules/
AddOns:
  Swatter, v3.4.6977 (SwimmingSeadragon)
  AucAdvanced, v3.4.6952 (SwimmingSeadragon)
  AucFilterBasic, v3.4.6941 (SwimmingSeadragon)
  AucScanData, v3.4.6908 (SwimmingSeadragon)
  AucStatHistogram, v3.4.6928 (SwimmingSeadragon)
  AucStatiLevel, v3.4.6913 (SwimmingSeadragon)
  AucStatPurchased, v3.4.6910 (SwimmingSeadragon)
  AucStatSimple, v3.4.6911 (SwimmingSeadragon)
  AucStatStdDev, v3.4.6912 (SwimmingSeadragon)
  AucUtilFixAH, v3.4.6914 (SwimmingSeadragon)
  BagBrother, v
  Bagnon, v10.2.18
  BeanCounter, v3.4.6983 (SwimmingSeadragon)
  Clique, vv3.5.6-release
  DBMCore, vf4d2083
  DBMStatusBarTimers, vf4d2083
  Details, v#Details.20240204.12256.155
  DetailsCompare2, v
  DetailsEncounterDetails, v
  DetailsRaidCheck, v
  DetailsStreamer, v
  DetailsTinyThreat, v
  DetailsVanguard, v
  ElvUI, v13.56
  ElvUILibraries, v13.56
  ElvUIOptions, v13.56
  Enchantrix, v3.4.6979 (SwimmingSeadragon)
  EnchantrixBarker, v3.4.6917 (SwimmingSeadragon)
  Informant, v3.4.6980 (SwimmingSeadragon)
  ItemRack, v3.77
  Questie, v9.3.3
  SlideBar, v3.4.6919 (SwimmingSeadragon)
  Stubby, v3.4.6920 (SwimmingSeadragon)
  TomTom, vv3.5.5-release
  WeakAuras, v5.9.2
  Whoimi, v1.0.0
  BlizRuntimeLib v1.15.1.53181(US) <enUS>
  (ck=4c8)
```

### Verification Steps

- [X] I have read the [Changelog](https://github.com/tukui-org/ElvUI/blob/development/CHANGELOG.md) and verified [ElvUI](https://tukui.org/elvui) is up to date by checking **/estatus**
- [X] I have verified my issue doesn't exist in the Default UI or the [Issue Tracker](https://github.com/tukui-org/ElvUI/issues).
- [ ] I have not read any of these steps, please close my issue when you see it.
- [X] I have done the Troubleshooting Steps and included the requested screenshots, which show the issue and status panel.

**Comment by 1Botanica1** [comment]
You didn't do the troubleshooting as requested. 

This buffs and debuffs is a blizzard issue - waiting on a fix from blizzard. 
From discord - The player buffs and debuffs (near the minimap) - currently this will be always off until Blizzard addresses an issue with the Secure Headers for Auras

**Comment by kodewdle** [comment]
fixed in 13.57
